includes/class-database-cleanup.php

```
@@ -308,13 +441,27 @@ public static function auto_clean( $settings ) {
 		$max_age = isset( $settings['dbRevMaxAge'] ) ? (int) $settings['dbRevMaxAge'] : 30;
 		$keep    = isset( $settings['dbRevKeepLatest'] ) ? (int) $settings['dbRevKeepLatest'] : 5;
 
-		self::clean_revisions_advanced( $max_age, $keep );
-		self::clean_auto_drafts();
-		self::clean_trashed_posts();
-		self::clean_spam_comments();
-		self::clean_trashed_comments();
-		self::clean_expired_transients();
-		self::clean_orphan_postmeta();
+		if ( false === self::clean_revisions_advanced( $max_age, $keep ) ) {
+			Log::add( 'Auto cleanup failed: clean_revisions_advanced' );
```

PHPMD (2.15.0)

[error] Avoid using static access to class 'PerformanceOptimise\Inc\Log' in method 'auto_clean'. (undefined)

(StaticAccess)